http/rest-client-reactive/src/main/java/io/quarkus/ts/http/restclient/reactive/files/FileResource.java

```diff
@@ -30,14 +30,14 @@ public class FileResource {
     private final OsUtils utils;
     private final List<File> deathRow = new LinkedList<>();
 
-    public FileResource(@ConfigProperty(name = "client.filepath") Optional<String> folder) {
+    public FileResource(@ConfigProperty(name = "client.filepath", defaultValue = "/tmp") Optional<String> folder) {
         utils = OsUtils.get();
         file = folder
                 .stream()
                 .map(existing -> java.nio.file.Path.of(existing).resolve("server.txt").toAbsolutePath())
                 .peek(path -> utils.createFile(path, BIGGER_THAN_TWO_GIGABYTES))
                 .map(java.nio.file.Path::toFile)
-                .findFirst().orElse(null);
+                .findFirst().orElseThrow(() -> new RuntimeException("server.txt creation failed!."));
```

Comment:
I was debugging an error and I miss a "configProperty" default value and throw an exception in case that `server.txt` is not created.